.github/workflows/sub_buildWindowsConda.yml

```diff
@@ -0,0 +1,140 @@
+      CCACHE_LOGFILE: C:/logs/ccache.log
+      ## Have to use C:\ because not enough space on workspace drive
+      builddir: C:/FC/build/
+      ccachebindir: C:/FC/ccache/
+      logdir: C:/logs/
+      reportdir: C:/report/
+      reportfilename: ${{ inputs.artifactBasename }}-report.md
+    outputs:
+      reportFile: ${{ steps.Init.outputs.reportFile }}
+
+    steps:
+      - name: Checking out source code
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+      - name: Setup Miniconda
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          activate-environment: .conda/freecad
+          environment-file: conda/conda-env.yaml
+          channels: conda-forge,defaults
+          channel-priority: true
+          miniforge-version: latest
+      - name: Cache conda
+        uses: actions/cache@v3
+        env:
+          # Increase this value to reset cache if conda/environment.devenv.yml has not changed
+          CACHE_NUMBER: 0
+        with:
+          path: "C:\\Users\\runneradmin\\conda_pkgs_dir"
+          key:
+            ${{ runner.os }}-conda-${{ env.CACHE_NUMBER }}-${{ hashFiles('conda/environment.devenv.yml') }}
+      - name: Install FreeCAD dependencies
+        run: |
+          conda config --add envs_dirs $PWD/.conda
+          mamba-devenv -f conda/environment.devenv.yml
+      - name: Make needed directories, files and initializations
+        id: Init
+        run: |
+          mkdir ${{ env.CCACHE_DIR }}
+          mkdir ${{ env.ccachebindir }}
+          mkdir ${{ env.builddir }}
+          mkdir ${{ env.logdir }}
+          mkdir ${{ env.reportdir }}
+          echo "reportFile=${{ env.reportfilename }}" >> $GITHUB_OUTPUT
+      - name: Get Ccache
+        uses: ./.github/workflows/actions/windows/getCcache
+        with:
+          ccachebindir: ${{ env.ccachebindir }}
+      - name: Restore compiler cache
+        uses: pat-s/always-upload-cache@v3
+        with:
+          path: ${{ env.CCACHE_DIR }}
+          key: FC-Windows-Conda-${{ github.ref }}-${{ github.run_id }}
+          restore-keys: |
+            FC-Windows-Conda-${{ github.ref }}-
+            FC-Windows-Conda-
+      - name: Print Ccache statistics before build, reset stats and print config
+        run: |
+          . $env:ccachebindir\ccache -s
+          . $env:ccachebindir\ccache -z
+          . $env:ccachebindir\ccache -p
+      - name: Configuring CMake
+        shell: cmd /C CALL {0}
+        run: >
+          conda\cmake.cmd --preset conda-windows-release -DFREECAD_USE_PCH:BOOL=OFF -DFREECAD_RELEASE_PDB:BOOL=OFF -DFREECAD_USE_CCACHE:BOOL=ON
+      - name: Compiling sources
+        shell: cmd /C CALL {0}
+        run: >
+          conda\cmake.cmd --build build\release
+      - name: Print Ccache statistics after build
+        run: |
+          . $env:ccachebindir\ccache -s
+      - name: C++ unit tests
+        if: false # Disabled because seems to not exist on Windows build
```

Comment:
is this still correct?

Reply:
This is copied over from the current Windows builds.  If this should be changed, I'd be happy to make the change.

I'll enable it and see.

Reply:
The C++ unit tests have [failed on Windows](https://github.com/oursland/FreeCAD/actions/runs/7065527459/job/19235714431#step:14:20), but succeeded on the [Windows on Conda](https://github.com/oursland/FreeCAD/actions/runs/7065527459/job/19235714621#step:14:22).

I also unfortunately discovered that [build failures on Ubuntu will be considered successful builds](https://github.com/oursland/FreeCAD/actions/runs/7065250572/job/19234889073#step:9:51650).  I believe this may be due to the [redirection in the build script](https://github.com/FreeCAD/FreeCAD/blob/main/.github/workflows/actions/linux/build/action.yml#L51), but I am not entirely certain.